README.md

```diff
@@ -38,14 +38,15 @@ Install the latest version of OMERO.server and OMERO.web and then:
     $ wget -O master.zip https://github.com/openmicroscopy/virtual-microscope/zipball/master
     $ unzip master.zip
     $ mv openmicroscopy-virtual-microscope-* virtual-microscope
+    $ ln -s /path/to/virtual-microscope /path/to/omero/lib/python/omeroweb/virtualmicroscope
```

Comment:
Based on https://github.com/openmicroscopy/virtual-microscope/pull/3, I would remove this and just say something about having it on PYTHONPATH.